apps/address-book/.gitignore

```diff
@@ -1,4 +1,30 @@
-node_modules
-build
+# dependencies
+/node_modules
+
+# testing
+/coverage
+coverage.json
+
+# production
+/build
+
+# development
 .cache
-dist
\ No newline at end of file
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# built assets
+/public/aragon-ui
+/public/script.js
+/public/script.map
+
+package-lock.json
```

Comment:
Same question for this. It seems helpful, but I'd prefer a repo-wide PR that implements this change separate from the significant effort of upgrading the Address Book contracts.